packages/account/src/Components/article/article.jsx

```diff
@@ -9,7 +10,9 @@ const Article = ({ className, title, descriptions }) => (
             {descriptions &&
                 descriptions.map((desc, idx) => (
                     <li key={idx}>
-                        <p className='account__article-text'>{desc}</p>
+                        <Text className='account__article-text' as='p' size='small'>
```

Comment:
Do we have `small` for text size? I think it should be `s` and be the default size for **Text component**

Reply:
You are right. Changing all...